apps/editing-toolkit/bin/verify-textdomain.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+set -Eeuo pipefail
+
+if [ ! -f "./editing-toolkit-plugin/newspack-blocks/synced-newspack-blocks/class-newspack-blocks.php" ] ; then
+	echo "Synced newspack-blocks directory doesn't exist. ETK needs to be built for this check to work."
+	exit 1
+fi
+
+if [ ! -f "./editing-toolkit-plugin/starter-page-templates/dist/starter-page-templates.min.js" ] ; then
+	echo "ETK's JS files have not been built. ETK needs to be built for this check to work."
+	exit 1
+fi
+
+set +e
+eslint_result=$(npx eslint-nibble --no-interactive --rule '@wordpress/i18n-text-domain' .)
```

Comment:
I greatly appreciate we're using `eslint-nibble`! It's much better and less fragile if we precisely fix only this rule and no others.

Reply:
great idea 👍